bundles/org.openhab.binding.openuv/src/main/java/org/openhab/binding/openuv/internal/handler/OpenUVReportHandler.java

```diff
@@ -70,24 +69,18 @@ public void initialize() {
 
         OpenUVConfiguration config = getConfigAs(OpenUVConfiguration.class);
 
-        String errorMsg = null;
-
         if (config.refresh != null && config.refresh < 3) {
```

Comment:
The compiler will not like this. If refresh can be null, it will warn about the usage later on.
Can refresh not be a default initialized integer, so not null at all?